src/makefile.standalone

```diff
@@ -1,7 +1,7 @@
 ifeq ($(strip $(CXX)),)
 CXX      = g++
-endif
-CXXFLAGS = -g $(INCLUDES) $(OPTFLAGS) $(CXXEXTRA) $(CXXPROF) $(WFLAGS) -MMD -std=c++11
+endif 
+CXXFLAGS = -g $(INCLUDES) $(OPTFLAGS) $(CXXEXTRA) $(CXXPROF) $(WFLAGS) -MMD -std=c++14 
 CFLAGS   = -g $(INCLUDES) $(OPTFLAGS) $(CCEXTRA) -MMD
```

Comment:
@bpbond and @crvernon, when I was having issues building stand-alone hector on my machine BBL had me change this line when clang errors were causing `make hector` to fail. It is not clear if this is universally needed or only needed by a specific set up. It was pretty difficult to figure out... should it be included as a commented out line instead? Or should it just be a personal Hector setting?

Reply:
1. I don't know.
2. I'd be inclined to has `c++14` be the default; this is over 6 years old now so it doesn't seem unreasonable to assume.